src/Sylius/Bundle/AdminBundle/templates/catalog_promotion/product_variant/index.html.twig

```diff
@@ -0,0 +1,3 @@
+{% extends '@SyliusAdmin/shared/crud/index.html.twig' %}
+
+{% set resource_name = 'catalog_promotion.product_variant' %}
```

Comment:
Is it necessary to handle this assignment on the index page? I'm curious why we can't just rely on the generic `@SyliusAdmin/shared/crud/index.html.twig`.

Reply:
Generic is `product_variant` because it represents a list of product variants. However, we want to specify that this is product variants in the catalog promotion section.